prow/scripts/provision-vm-and-start-kyma.sh

```diff
@@ -28,34 +24,31 @@ else
     LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=kyma-integration")
 fi
 
-echo "
-################################################################################
-# Creating a new instance named kyma-integration-test-${RANDOM_ID}
-################################################################################
-"
-
-gcloud compute instances create "kyma-integration-test-${RANDOM_ID}" \
-    --metadata enable-oslogin=TRUE --image debian-9-stretch-v20181011 \
-    --image-project debian-cloud --machine-type n1-standard-4 --boot-disk-size 20 "${LABELS[@]}"
+ZONE_LIMIT=${ZONE_LIMIT:-5}
+EU_ZONES=$(gcloud compute zones list --filter="name~europe" --limit="${ZONE_LIMIT}" | tail -n +2 | awk '{print $1}')
+
+for ZONE in ${EU_ZONES}; do
+    shout "Attempting to create a new instance named kyma-integration-test-${RANDOM_ID} in zone ${ZONE}"
+    gcloud compute instances create "kyma-integration-test-${RANDOM_ID}" \
+        --metadata enable-oslogin=TRUE \
+        --image debian-9-stretch-v20181011 \
+        --image-project debian-cloud \
+        --machine-type n1-standard-4 \
+        --zone "${ZONE}" \
```

Comment:
Please then remove the usage of `preset-gc-compute-envs` label from the job definition.

Reply:
I think this comment was supposed to be in a different issue :)

Reply:
Zone was given as an environment variable (`CLOUDSDK_COMPUTE_ZONE`) before. Since you specify it as a parameter, we no longer need to add `preset-gc-compute-envs` label to the job to have `CLOUDSDK_COMPUTE_ZONE` set.